frontend/src/Style/Flex.elm

```diff
@@ -0,0 +1,9 @@
+module Style.Flex exposing (itemsCenter, justifyBetween)
+
+
+justifyBetween =
+    "justify-between"
+
+
+itemsCenter =
```

Comment:
One of the goals with this refactor is to make more readable names (i.e. `Color.darkGreen` instead of `green-dark`). Looking ahead to a future where we define all of the possible values for `align-items`, maybe `centerItems` is more appropriate?

* `startItems` (weird)
* `endItems` (weird)
* `centerItems` (good)
* `stretchItems` (good)
* `baselineItems` (bad)

Or considering `align-content` exists and has similar values to `align-items`, we may want to fully disambiguate the two:

* `centerAlignItems`
* `centerAlignContent`